packages/date-picker/src/FilterDateRangePicker/FilterDateRangePicker.spec.tsx

```diff
@@ -15,6 +15,8 @@ const openFilter = async (): Promise<void> => {
 
 const FilterDateRangePickerWrapper = ({
   selectedRange,
+  status,
+  validationMessage,
```

Comment:
I think we should leave these as part of the spread props :)

Reply:
It causes a TS error otherwise

Reply:
Hmm...that's a problem though. We should be able to safely pass in these values without an error.

Reply:
It says we can't pass undefined to it

Reply:
<img width="906" alt="image" src="https://user-images.githubusercontent.com/48232362/220827444-edeb1aaa-4c07-4456-a140-845ef8afdfd7.png">